microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -1,75 +1,145 @@
 ---
-description: Learn how to add and manage users from your organization on to the Microsoft Edge program
-title: Register your organization for Microsoft Edge program on Partner Center
+description: Adding and managing users from your organization onto the Microsoft Edge program to help manage the Partner Center account.  Enable your other team members to publish Microsoft Edge extensions to the Microsoft Edge Add-ons website using your Partner Center account.
+title: Add users to the Microsoft Edge program
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/30/2021
+ms.date: 08/27/2021
 ms.topic: conceptual
 ms.prod: microsoft-edge
 keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer
 ---
+# Add users to the Microsoft Edge program
 
-# Add and manage users from your organization on to the Microsoft Edge program
+<!-- better? # Add users to your Partner Center account -->
+<!-- todo globally: "Microsoft Edge program", or other term? -->
 
-To add and manage users in the Microsoft Edge program to manage extensions, you can now associate your Partner Center account with your organization's Azure Active Directory.
+To help manage Microsoft Edge extensions, you can add more users to an existing Partner Center account.  To manage Microsoft Edge extensions, the Primary Owner of the Partner Center account must be a Microsoft account (MSA).
 
+A Microsoft account (MSA) is an Outlook.com, Live.com, or Hotmail.com account.  For an overview, see [Types of accounts related to publishing Microsoft Edge extensions](create-dev-account.md#types-of-accounts-related-to-publishing-microsoft-edge-extensions).
+
+
+<!-- ====================================================================== -->
+## Making sure you have a Partner Center account with a Microsoft account (MSA) as the Primary Owner
+
+To create a Partner Center account that can publish Microsoft Edge extensions, you must have a Microsoft account (MSA), either by creating one directly or by using your personal GitHub account credentials.  
+
+After your Partner Center account is able to publish Microsoft Edge extensions, you can link the Partner Center account to an Azure Active Directory (Azure AD) tenant.  An linked Active Directory tenant enables the added users to login to your Partner Center developer account by using their work accounts.
+
+Different programs on Partner Center require different types of accounts:
+
+*  The Microsoft Edge program (like the Windows developer program) requires a Partner Center _developer_ account.  A Partner Center developer account has a Primary Owner that's a Microsoft account (MSA).
+
+*  In contrast, Azure marketplace requires a Partner Center _commercial_ account.  (To enroll, the user has to login with their work account.)
```

Comment:
This looks good to me and I think we should keep this section (definition of commercial account) for future.